packages/components/src/LightBox/LightBox.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { LightBox } from ".";
+
+afterEach(cleanup);
```

Comment:
A test in here confirming that the `onRequestClose` gets fired when the user clicks close would be good.